### PR TITLE
Refactor other previews

### PR DESCRIPTION
**Pull request by akomiqaia**
## Describe your changes
- update previews for button component
- add a new feature to preview components to highlight code parts
- add tooltip previews

### CODE REVIEW

File: apps/for-everyone-website/src/styles/custom.css [hunk 1]
```diff
@@ -39,3 +39,12 @@ img {
...
 }
+
+
+[data-o3-theme="inverse"] .component-preview {
+	background-color: var(--o3-color-use-case-page-inverse-background);
+}
+
+[data-o3-theme="mono"] .component-preview {
+	background-color: white;
+}
```
Comment (notlee) [inline apps/for-everyone-website/src/styles/custom.css:50]: Mono on paper is a thing, so I think we can delete this?

File: apps/for-everyone-website/src/styles/custom.css [hunk 2]
```diff
@@ -39,3 +39,12 @@ img {
 .blog-post figcaption {
 	font-size: var(--sl-text-xs);
 }
+
+
+[data-o3-theme="inverse"] .component-preview {
+	background-color: var(--o3-color-use-case-page-inverse-background);
+}
```
Comment (notlee) [inline apps/for-everyone-website/src/styles/custom.css:46]: Yay o3 theming! 🕺 

The dot is quite prominent though, maybe distracting. What do you think about making it darker or adding opacity to bring it closer to slate?

File: apps/for-everyone-website/src/content/docs/sustainable-views/components/tooltip.mdx [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+2. <strong>Text</strong>: The main content providing detailed explanation or
+   information.
+3. <strong>Close button (×)</strong>: This button dismisses the tooltip. Once
+   closed, the tooltip should not reappear for the user.
+4. <strong>Arrow</strong>: A directional arrow that can point towards any side,
+   indicating the source of the information or context.
+5. <strong>Container</strong>: The encompassing frame that holds the message
+   text and ensures it's distinct from other UI elements.
+
+<TooltipPreview component="onboarding" brand="sustainable-views" />
```
Comment (notlee) [inline apps/for-everyone-website/src/content/docs/sustainable-views/components/tooltip.mdx:43]: Proper preview for tooltips hype!

File: apps/for-everyone-website/src/components/button/ButtonPaginationPreview.astro [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+import * as InverseComponent from './preview/ButtonPagination-inverse';
+import Preview from '../utils/Preview.astro';
+import FigmaSbLinks from '../utils/FigmaSbLinks.astro';
+
+const {theme} = Astro.props;
+
+const Component = theme === 'inverse' ? InverseComponent : StandardComponent;
+const figma =
+	'https://www.figma.com/file/nSD9N3r4MiMpv9pA6F4Dpy/Pagination?type=design&node-id=5463-1742&mode=design';
+const storybook = `https://o3.origami.ft.com?path=/story/core-o3-button--pagination&args=theme:${theme}`;
```
Comment (notlee) [inline apps/for-everyone-website/src/components/button/ButtonPaginationPreview.astro:12]: Could we take the opportunity here to link to the correct brands?

File: apps/for-everyone-website/src/components/tooltip/preview/OnboardingTooltip.tsx [hunk 5]
```diff
@@ -0,0 +1,37 @@
+import {TooltipOnboarding} from '@financial-times/o3-tooltip/esm';
+import {Button} from '@financial-times/o3-button/esm';
+import './demo-styles.css';
+function OnboardingTooltipPreview() {
+	return (
+		// <preview>
```
Comment (notlee) [inline apps/for-everyone-website/src/components/tooltip/preview/OnboardingTooltip.tsx:6]: Need to move `<preview>` here :)

File: apps/for-everyone-website/src/components/button/ButtonPaginationPreview.astro (hunk 4 above)
Comment (akomiqaia) [reply apps/for-everyone-website/src/components/button/ButtonPaginationPreview.astro:12]: how do we get brand here? do we create paterns inside brand folders?

**Review by notlee — APPROVED** [review]

**Pull request merged**